src/components/Widgets/RelationControl.js

```diff
@@ -81,10 +89,26 @@ class RelationControl extends Component {
     return suggestion.data[valueField];
   };
 
+  getSuggestionDisplayValue = (suggestion) => {
+    const { field } = this.props;
+    const displayField = field.get('displayField');
+    return suggestion.data[displayField];
+  };
+
   renderSuggestion = (suggestion) => {
     const { field } = this.props;
-    const valueField = field.get('valueField');
-    return <span>{suggestion.data[valueField]}</span>;
+    const displayField = field.get('displayField');
+    return <span>{suggestion.data[displayField]}</span>;
+  };
+
+  renderInputComponent = (inputProps) => {
+    const { value, displayValue, id, ...otherInputProps } = inputProps;
+    return (
+      <div>
+        <input {...otherInputProps} value={displayValue} />
+        <input type="hidden" id={id} value={value} />
```

Comment:
Need some explanation on this too.

Reply:
The single input field was replaced by two input fields.

The first is the visible one, displaying the `displayValue` to the user. `{...otherInputProps}` is needed to [preserve the field accessibility](https://github.com/moroshko/react-autosuggest#renderInputComponentProp).

The second stores the `value`, in order for it to be saved.